### DISCUSSION TITLE
Raspberry Pi Zero2w hàngt sich auf

### DISCUSSION THREAD

**Discussion by Clarion30** [camueller/SmartApplianceEnabler#544] [Installation und Betrieb]
Hallo, bin jetzt vom Pi 3 auf eine Zero 2 umgezogen, mit USB Hub sowie USB Netzwerkadapter außerdem hängt noch ein Modbus Adapter dran.
Der 3 Pi lief ohne Probleme seit Monaten.
Der Zero 2 hängt sich ständig auf und blinkt nur, keinen Zugriff auf per SSH.
Nach dem Hardreset läuft er ohne Probleme ein paar Stunden.
Neuinstallation brachte keine Abhilfe
Standby Funktion des WLAN habe ich schon deaktiviert, hat nichts gebracht.
Jemand noch eine Idee ?
Danke für die Hilfe

Gruß Christoph

**Comment by camueller** [comment]
HIer nochmal der Hinweis, dass der Pi Zero nicht supported ist: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Raspberry_DE.md

**Reply by Clarion30** [reply]
Moin,bin jetzt vom Zero ausgegangen nicht vom Zero 2 weil der ja von der Spezifikationen ja doch besser ist wie der 2B ?
Also werden keine Zeros unterstütz ?

**Comment by camueller** [comment]
Ich habe die Specs nicht direkt verglichen, aber in der Verangeneheit war die CPU der Pi Zero zu schwach. Wenn es so ist, wie Du schreibst, sollte es ja keine Probleme geben. Ich habe immer noch einen Pi 2B im produktiven Einsatz.

**Reply by Clarion30** [reply]
ist auch eine 64-bit ARM Cortex-A53 CPU. wie bei pi3 aber weniger Ram nur 512.
Aber sind trotzdem noch unterschiede.
Werde mal weiter testen mal mit aktiven Hub und Display Ausgang.

**Reply by camueller** [reply]
512 MB RAM ist zu wenig! So schlank ist die Java nicht.

**Reply by Clarion30** [reply]
hab ich mir schon gedacht, werde mal swapping ausprobieren. Werde dann bei 3er bleiben.

**Reply by raldie** [reply]
Hallo Clarion30,
ich habe einen Zero 2W in den letzten 3 Wochen im Livebetrieb getestet, er läuft problemlos mit dem SAE (v2.3.0).
Eine komplette manuelle Installation, inkl. Webmin ist dafür aufgesetzt worden.
Es wurden alle meine 7 Geräte einmal per HTTP angebunden und eine Woche getestet, danach die MQTT-Version einkopiert und getestet. Alles lief genauso problemlos wie mein PI 3B in den letzten 1,5 Jahren.
Im Webmin wurden in der Standardkonfiguration ca. 44% Hauptspeicher belegt und 87MB Auslagerungsdatei.
Wie bei allen meinen Raspberrys verwende ich allerdings keine SD-Karten im Betrieb, sondern einen guten USB-Stick, bzw. eine SSD als Boot/Speichermedium.
Die Problem könnte durchaus auch an der SD Karte liegen, am PI Zero 2W sollte es nicht liegen.

**Reply by Clarion30** [reply]
Ja danke für die Info hab jetzt mal die manuelle Installation aufgespielt, morgen abend weiß ich mehr :)

**Reply by Clarion30** [reply]
Also ich kann jetzt schon sagen das es länger läuft mit der manuellen Installation als mit der automatischen Installation. Hätte jetzt gedacht das die manuelle Installation genauso ist wie mit der automatischen. 
Anscheinend sind da doch  Unterschiede.
Nochmal danke für die Hilfe

**Comment by Clarion30** [comment]
Falls es interressiert nach manueller Installation läuft der Zero2 seit Mitte Juli durch !!
